libraries/classes/Util.php

```diff
@@ -3652,7 +3651,7 @@ public static function getGISFunctions(
             'type' => 'int',
         ];
 
-        $geom_type = trim(mb_strtolower((string) $geom_type));
+        $geom_type = mb_strtolower(trim((string)$geom_type));
```

Comment:
The coding standard requires 1 space after the cast statement.

```suggestion
        $geom_type = mb_strtolower(trim((string) $geom_type));
```

Reply:
Sorry. Fixed.